gensim/models/word2vec.py

```diff
@@ -459,6 +459,7 @@ def __init__(
         self.total_train_time = 0
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
+        self.size = size
```

Comment:
This new instance variable seems unnecessary; better to consult existing `layer1_size`.